### ISSUE TITLE
Production

### ISSUE THREAD

**Issue by rmartins90** [zhiyand/vue-features-loader#1]
Hi @zhiyand. Was this package ever used in production environment?

**Comment by zhiyand** [comment]
@rmartins90 Nope. I've only used it for my side projects. It is never been battle tested. :P
